compiler/stack.yaml

```diff
@@ -28,18 +28,9 @@ resolver: lts-7.24
 packages:
 - .
 
-flags:
-  bond:
-    warningsAsErrors: true
```

Comment:
This was needed so that we could build bond as part of CI builds treating warnings as errors but not to force that on everyone else who may use the Hackage package. Is there a replacement to this functionality?

Reply:
I've added it in the package.yaml as, I thought, `-Wall -Werror` are  recommended. However, I understand we want to keep it configurable. I will make it so in the next iteration.